tests/test_data/littlesunnykitchen.com/littlesunnykitchen.json

```
@@ -0,0 +1,67 @@
+{
+  "host": "littlesunnykitchen.com",
+  "canonical_url": "https://littlesunnykitchen.com/mexican-chicken-marinade/",
+  "site_name": "Little Sunny Kitchen",
+  "author": "Diana",
+  "language": "en-US",
+  "title": "Mexican Chicken Marinade",
+  "ingredients": [
+    "2 tablespoons olive oil",
+    "3 cloves garlic (minced)",
+    "1 tablespoon fresh lime juice",
+    "2 teaspoons chili powder",
+    "1 teaspoon smoked paprika",
+    "1 teaspoon dried oregano",
+    "1 teaspoon ground cumin",
+    "¼ teaspoon ground cinnamon (optional)",
+    "1 teaspoon salt",
+    "1-2 pounds chicken (breasts, cutlets, thighs, or drumsticks)"
+  ],
+  "ingredient_groups": [
+    {
+      "ingredients": [
+        "2 tablespoons olive oil",
+        "3 cloves garlic (minced)",
+        "1 tablespoon fresh lime juice",
+        "2 teaspoons chili powder",
+        "1 teaspoon smoked paprika",
+        "1 teaspoon dried oregano",
+        "1 teaspoon ground cumin",
+        "¼ teaspoon ground cinnamon (optional)",
+        "1 teaspoon salt",
+        "1-2 pounds chicken (breasts, cutlets, thighs, or drumsticks)"
+      ],
+      "purpose": null
+    }
+  ],
+  "instructions": "To a Ziploc bag, add the chicken followed by the wet ingredients and the spices.\nSeal the bag, and toss the chicken so that the chicken is well coated in the marinade.\nPlace in the fridge and allow to marinate for 2-24 hours.\nRemove the chicken from the bag, and grill it or bake it until the internal temperature of the chicken reaches 165°F/74°C with an Instant read thermometer.",
+  "instructions_list": [
+    "To a Ziploc bag, add the chicken followed by the wet ingredients and the spices.",
+    "Seal the bag, and toss the chicken so that the chicken is well coated in the marinade.",
+    "Place in the fridge and allow to marinate for 2-24 hours.",
+    "Remove the chicken from the bag, and grill it or bake it until the internal temperature of the chicken reaches 165°F/74°C with an Instant read thermometer."
+  ],
+  "total_time": 130,
+  "prep_time": 10,
+  "category": "Main Course",
+  "yields": "6 servings",
+  "cuisine": "American,Mexican",
+  "nutrients": {
+    "calories": "49 kcal",
+    "carbohydrateContent": "2 g",
+    "proteinContent": "1 g",
+    "fatContent": "5 g",
+    "saturatedFatContent": "1 g",
+    "sodiumContent": "400 mg",
+    "fiberContent": "1 g",
+    "sugarContent": "1 g",
+    "servingSize": "1 serving"
+  },
+  "ratings": 4.95,
+  "ratings_count": 57.0,
```

Nitpicky: I think this should be an integer value:

```suggestion
  "ratings_count": 57,
```

---

I was expecting an integer as well, but the `scraper` is returning a float in my testing.

```py
In [1]: from recipe_scrapers import scrape_me

In [2]: scraper = scrape_me("https://littlesunnykitchen.com/mexican-chicken-marinade/")

In [3]: scraper.ratings_count()
Out[3]: 57.0
```

---

Ok - and I can confirm that the [`schema.org` definition of `AggregateRating`](https://schema.org/AggregateRating) type that we _usually_ retrieve this field from defines `ratingCount` as an integer - so this seems like a bug that we should fix.  I'll file a bugreport for that in a few moments.

(I had noticed that we should standardise the type in our test JSON files as a task in #1173.. but changing the runtime return type is a separate issue)

---

Thanks for raising this - it's now tracked by issue #1175.